filebeat/_meta/fields.common.yml

```diff
@@ -95,6 +95,11 @@
     - name: source_ecs
       type: group
       fields:
+      - name: domain
```

Comment:
Looks like the wrong place for it under `source_ecs`?

Reply:
Not sure what you mean. We do have `source.domain` defined in ECS.

Until #8983 is merged, this is the place to define the source's domain :-)